plugin/pkg/scheduler/schedulercache/node_info.go

```diff
@@ -70,9 +70,9 @@ type Resource struct {
 	EphemeralStorage int64
 	// We store allowedPodNumber (which is Node.Status.Allocatable.Pods().Value())
 	// explicitly as int, to avoid conversions and improve performance.
-	AllowedPodNumber  int
-	ExtendedResources map[v1.ResourceName]int64
-	HugePages         map[v1.ResourceName]int64
+	AllowedPodNumber int
+	// OIRs and HugePages
```

Comment:
I would just write ScalarResources to cover other scalar resources we add in the future. Besides, OIRs is a bit vague and are deprecated.